src/main/java/seedu/weme/ui/CommandBox.java

```diff
@@ -43,6 +55,68 @@ public CommandBox(CommandExecutor commandExecutor, CommandPrompter commandPrompt
         commandTextField.addEventFilter(KeyEvent.KEY_PRESSED, this::handleTabPress);
     }
 
+    /**
+     * Handles the key press event, {@code keyEvent}.
+     */
+    @FXML
+    private void handleKeyPress(KeyEvent keyEvent) {
+        switch (keyEvent.getCode()) {
+        case UP:
+            // As up and down buttons will alter the position of the caret,
+            // consuming it causes the caret's position to remain unchanged
+            keyEvent.consume();
+            handleLikeByKeyPress(KEYPRESS_UP);
+            break;
+        case DOWN:
+            keyEvent.consume();
+            handleLikeByKeyPress(KEYPRESS_DOWN);
+            break;
+        case LEFT:
+            keyEvent.consume();
+            handleIndexToggleByKeyPress(KEYPRESS_LEFT);
+            break;
+        case RIGHT:
+            keyEvent.consume();
+            handleIndexToggleByKeyPress(KEYPRESS_RIGHT);
+            break;
+        default:
+            // let JavaFx handle the keypress
+        }
+    }
+
+    /**
+     * Changes index in the command box with key press to enable fast liking.
+     */
+    private void handleIndexToggleByKeyPress(String keyPress) {
```

Comment:
You can't move left and right to edit a part of the command if the only argument is a number, this might be an issue, but it's expected as well. since left and right should update the indexes accordingly

This method also works as long as the arguments is just a number, so might want to change the doc. Will need to think about when this feature should be active

Reply:
I can limit it to like first. It only works if the input only consists of a valid command keyword and a number (index or not). Might still be useful for other command. What do the rest think?